### PR TITLE
WIP WIP: Fix #15132: Integrate table to I based commands

### LINKED ISSUE #15132: Use table api in all the `i` subcommands
Screenshot 2019-09-26 at 23 47 03

### PR DESCRIPTION
**Pull request by deepakchethan**
Integrated with:
* is
* ia
* iS
* ir
* iz

### CODE REVIEW

File: libr/util/table.c [hunk 1]
```diff
@@ -676,6 +676,10 @@ R_API bool r_table_query(RTable *t, const char *q) {
 }
```
Comment (radare) [inline libr/util/table.c]: empty line

**Comment by radare** [comment]
ping

**Comment by radare** [comment]
please rebase

**Comment by radare** [comment]
ping, cant wait to get this thing merged. will be good to have some updates or at least enumerate what's missing

**Comment by deepakchethan** [comment]
Lets merge this and i'll do the rest in another pr

File: libr/util/table.c (hunk 1 above)
Comment (deepakchethan) [reply libr/util/table.c]: Removed it

**Comment by radare** [comment]
Can you rebase and make it green?

**Comment by radare** [comment]
Some observations:

Dunno where this arch/bits info comes from :D 

Screenshot 2019-10-09 at 11 04 25

null is not our friend, i would suggest to use NONE instead of (null)

Screenshot 2019-10-09 at 11 04 51

Support align to right:

Screenshot 2019-10-09 at 11 05 37

we may probably want to remove the `----` line or make it as long as the longest line below

Screenshot 2019-10-09 at 11 06 01

this test fails hard: [31m[XX][39m db/formats/pe/pe [33mpe32 imported functions (ii)[39m  1666
 (lol at 0x08llx)
Screenshot 2019-10-09 at 11 07 58

Screenshot 2019-10-09 at 11 07 34

**Pull request merged**